src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/components/_icons.scss

```diff
@@ -42,7 +42,10 @@
   @if ($responsive: true) {
     @include breakpoint(sm) {
       background-size: $sml-size;
+      width: 32px;
```

Comment:
WE should make these variables. since they are icons that might be used elsewhere.